src/pegen/python_generator.py

```diff
@@ -214,6 +217,16 @@ def visit_Name(self, node: ast.Name) -> Set[str]:
         return {node.id}
 
 
+class RuleProperties(NamedTuple):
```

Comment:
this can be removes alongside is_loop1, as it then would not really add anything
although I still think it's nicer